<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>label_img_header_dialog</class>
 <widget class="QDialog" name="label_img_header_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_printer_MMwidth_sub">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_printPage">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>printPage</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_printer_MMwidth">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>MM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_printer_MMheight_add">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_key">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>250</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_top">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>up</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_right">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>right</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_left">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>left</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_down">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>down</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_scale_minus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>scale-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_scale_plus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>scale+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_selectAll">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>selectBatch</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_printer_MMwidth_add">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>251</width>
     <height>391</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>249</width>
      <height>389</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_check_list">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_printer_MMheight_sub">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_design">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>resolve</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>741</width>
     <height>181</height>
    </rect>
   </property>
   <property name="plainText">
    <string>printerName:Gprinter GP-3150TN
size:40x30
{&quot;desc&quot;:&quot;&quot;,&quot;font_pixel&quot;:30,&quot;list&quot;:[&quot;APPLE&quot;],&quot;name&quot;:&quot;item_name&quot;,&quot;type&quot;:&quot;text&quot;,&quot;x&quot;:10,&quot;y&quot;:12}
{&quot;desc&quot;:&quot;&quot;,&quot;font_pixel&quot;:23,&quot;list&quot;:[&quot;UHY,MOH&quot;],&quot;name&quot;:&quot;item_costumize&quot;,&quot;type&quot;:&quot;text&quot;,&quot;x&quot;:10,&quot;y&quot;:82}
{&quot;desc&quot;:&quot;No.:&quot;,&quot;font_pixel&quot;:23,&quot;list&quot;:[&quot;1002&quot;],&quot;name&quot;:&quot;serial_no&quot;,&quot;type&quot;:&quot;text&quot;,&quot;x&quot;:10,&quot;y&quot;:112}
{&quot;desc&quot;:&quot;Total:&quot;,&quot;font_pixel&quot;:23,&quot;list&quot;:[&quot;12.36&quot;],&quot;name&quot;:&quot;item_price&quot;,&quot;type&quot;:&quot;text&quot;,&quot;x&quot;:10,&quot;y&quot;:142}
{&quot;desc&quot;:&quot;&quot;,&quot;font_pixel&quot;:23,&quot;list&quot;:[&quot;EPIVERISON&quot;],&quot;name&quot;:&quot;shop_name&quot;,&quot;type&quot;:&quot;text&quot;,&quot;x&quot;:10,&quot;y&quot;:172}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>10</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_printer_MMheight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>MM</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_printAll">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>printAll</string>
   </property>
  </widget>
  <widget class="label_img_design_view" name="design" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>331</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>label_img_design_view</class>
   <extends>QWidget</extends>
   <header>label_img_design_view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
